### PR TITLE
Automatic index

### PR DESCRIPTION
**Pull request by fabienrohrer**
Preview: https://www.cyberbotics.com/doc/guide/robotis-op2?version=feature-automatic-index

### CODE REVIEW

**Comment by fabienrohrer** [comment]
👍

File: js/viewer.js [hunk 1]
```diff
@@ -336,6 +336,50 @@ function setUpBlogStyleIfNeeded() {
...
 
+function createIndex(view) {
+  // Note: the previous index is cleaned up when the parent title is destroyed.
+
+  // Get all the view headings.
+  var headings = [].slice.call(view.querySelectorAll('h1, h2, h3, h4'));
+
+  // Do not create too small indexes.
+  var content = document.querySelector('#content');
+  console.log(content.offsetHeight);
```
Comment (omichel) [inline js/viewer.js]: Is that a debug statement? If so, I would remove it.

**Comment by fabienrohrer** [comment]
Finally I moved the index above the first title, it seems more readable to me.

**Comment by fabienrohrer** [comment]
👍

**Pull request merged**